### ISSUE TITLE
Is dependencies possible?

### ISSUE THREAD

**Issue by loaibassam** [namespace-ee/react-calendar-timeline#806]
Are we able to connect items with other items? like shown on the screenshot below:

![image](https://user-images.githubusercontent.com/4785390/109605392-89802e80-7b3e-11eb-8cc0-700c9d2ec036.png)

**Comment by gprpaes** [comment]
Hi, any updates on this? I also would like to know if its possible to connect items with others.

Thanks in advance,
Guilherme.

**Comment by menasiephrem** [comment]
I don't think this is natively supported in the library. But we used another [library](https://github.com/Eliav2/react-xarrows) with  this one to show some kinda of dependency. I don't know if this helps. 

<img width="1327" alt="Screen Shot 2021-03-27 at 3 47 32 PM" src="https://user-images.githubusercontent.com/19403083/112721285-1cc93b80-8f14-11eb-9794-c503fcad2aff.png">

**Comment by gprpaes** [comment]
Thanks, I'll check it out!

**Comment by bwka1988** [comment]
@menasiephrem thanks for your comment. I'm new at react. Can you write a short example how you can implement it? I have both libraries imported but I don't know how to combine both. Would appreciate an answer. Thanks!

**Comment by developer-ocansey** [comment]
Closing as part of issue triage. This issue was created before 2023 and has had no recent activity. If you're still experiencing this on v0.30+, please open a new issue with a minimal reproduction.
